easybuild/main.py

```diff
@@ -618,6 +619,13 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None, pr
     (build_specs, _log, logfile, robot_path, search_query, eb_tmpdir, try_to_generate,
      from_pr_list, tweaked_ecs_paths) = cfg_settings
 
+    # compare running Framework and EasyBlocks versions
+    if EASYBLOCKS_VERSION == UNKNOWN_VERSION:
+        print_msg("Unable to determine EasyBlocks version, so we'll assume it is not different from Framework")
```

Comment:
This should be a warning?

Reply:
0a5c99f

Reply:
Printing something here is problematic, so we'll log it as a warning instead.

The most likely place for this to happen is running the framework unit tests without an easyblocks installation. If you try to use EB in that case then you'll fail really quickly.

The only other case I can think of is if something breaks the version lookup of the blocks. That is a rare enough possibility that I think we should ignore.